crates/bevy_render/Cargo.toml

```diff
@@ -62,12 +63,16 @@ image = { version = "0.24", default-features = false }
 codespan-reporting = "0.11.0"
 # `fragile-send-sync-non-atomic-wasm` feature means we can't use WASM threads for rendering
 # It is enabled for now to avoid having to do a significant overhaul of the renderer just for wasm
-wgpu = { version = "0.18", features = [
+wgpu = { version = "0.19", default-features = false, features = [
+  "wgsl",
+  "dx12",
+  "metal",
   "naga",
+  "naga-ir",
```

Comment:
Why was naga_ir added?

Reply:
It doesn't compile without it. `wgpu::ShaderSource::Naga` is gated behind it, and we use that when compiling shaders in the `ShaderCache`.